src/js/components/Marathon.jsx

```diff
@@ -371,7 +421,7 @@ var Marathon = React.createClass({
   getAppPage: function () {
     var activeApp = this.state.collection.get(this.state.activeAppId);
     if (!activeApp) {
-      return;
+      return null;
```

Comment:
I don't want to set 'null' explicit here, it's meant to be undefined.
Can we set 
```"consistent-return": 0```
in the .eslintrc ?